### ISSUE TITLE
Newbie questions

### ISSUE THREAD

**Issue by CCGLLC-knc** [iquidus/explorer#233]
Just installed this for a new alt-coin.  Overall, its seems to be working fine - blocks are being displayed, as is hashrate, coin supply, etc.  Searching is working great!

I noticed that an txcount greater than 100 is ignored.  I can live with 100 but curious if there is a way to get past that?

I'm running this on a 56 core (28 physical, 56 hyperthreaded) dual xeon.  As documented, I expected to see 56 cluster processes, but seeing over 400...  what am I missunderstanding?  Is it possible to set it to running something less, like 8?

Is there a bit more documentation on the coin supply calculation choices?  I'm running a bitcoin source derivative PoW coin and don't understand the advantages/disadvantages of each option.  Currently using COINBASE since that flagged for Pow coins.  Presume that is referring to the base transaction in every block, not the website <smile>.

For some reason, the getreward api is dying with a "method not found".  Presume that is a deficiency in my alt-coin code?  Alas, I'm not seeing a "getreward" method in any wallet I try, including Bitcoin Core ones.  What have I done wrong?  "getsupply" also suffers from this,  Console error below.

`{ Error: Method not found
    at /store/explorer/explorer/node_modules/bitcoin/lib/jsonrpc.js:99:19
    at Array.forEach (<anonymous>)
    at IncomingMessage.<anonymous> (/store/explorer/explorer/node_modules/bitcoin/lib/jsonrpc.js:96:15)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9) code: -32601 }
`

All of the extended APis return a "This method is restricted" response.  How do fix?  Should I fix???

Beyond that, implementation was incredibly easy.  Thank you!!!

**Comment by iquidus** [comment]
you can change the tx limit here: https://github.com/iquidus/explorer/blob/master/settings.json.template#L132 

it will require reindexing if this is increased.

How the cluster works is it essientually runs an instance of the explorer on each core and load balances between them. If you would like a fixed amount of instances you can change this line

https://github.com/iquidus/explorer/blob/master/bin/cluster#L22 

var cpuCount = 8; 

for example.

for supply on a pure PoW chain COINBASE is the correct setting. correct, its not a reference to the site, its a term used for where newly minted coins come from (block rewards).

you can derestrict daemon api calls here: https://github.com/iquidus/explorer/blob/master/app.js#L20

note: getreward is specific to heavycoin (HVC) and its forks, as are some others.

**Comment by CCGLLC-knc** [comment]
Thanks.  I saw the settings for tx limit, but did not know I needed to reindex afterwards.  Will do.  Hmm, reindexed and still only showing 100 entries.  I can easily live with that.

Also thanks for the cpuCount.  I just set that to actual / 4 in order to leave some cores free for other work.

So set Heavy to false.  Thanks for that explanation.

Noticed a lot of dangling cluster processes hanging around even after I've done an "npm stop".  Like I'm up to 459 of them.  Is it legal to stop via Ctrl-C on the console or must I use "npm stop"?  Note:  "killall node" cleaned everything up.
